src/app/static/src/js/utils/types.ts

```diff
@@ -53,7 +54,7 @@ export interface WorkflowReportWithParams {
     params?: Record<string, string>
 }
 
-export interface Error {
+export interface Errors {
```

Comment:
I think a better name for this might be `ErrorResponse` since it does represent the whole response which might contain some errors. As opposed to `Error` which is just an individual error within the response. In fact although it's marked here as `any[]` I think `errors` here could be `Error[]` 

Oh, and I've just seen that you've added `Response` below, which mostly does away with the need for this type since it's a more fully specified version of it except that this has the additional `response` property at the top level - I think this is because this actually represents an AxiosResponse which keeps the actual response data in `response`. So arguably you could replace this with something like `export type ErrorResponse = AxiosResponse<Response, any>` to make that more explicit. Except that's kind of confusing too since it might not be a response in error.... but that was a flaw of the previous type as well!

Reply:
> I think a better name for this might be `ErrorResponse` since it does represent the whole response which might contain some errors. As opposed to `Error` which is just an individual error within the response. In fact although it's marked here as `any[]` I think `errors` here could be `Error[]`
> 
> Oh, and I've just seen that you've added `Response` below, which mostly does away with the need for this type since it's a more fully specified version of it except that this has the additional `response` property at the top level - I think this is because this actually represents an AxiosResponse which keeps the actual response data in `response`. So arguably you could replace this with something like `export type ErrorResponse = AxiosResponse<Response, any>` to make that more explicit. Except that's kind of confusing too since it might not be a response in error.... but that was a flaw of the previous type as well!

This was just a rename and we had two Error type with the same name but I will rename as ErrorReponse

Reply:
As I say, just making it a rename is a bit non-DRY because you've now added that `Response` type too, which is essentially what that old `Error` type was (but much more fully specified - they're referring it the same things though), except it had an additional wrapper level. Hence my suggestion of `AxiosResponse<Response, any>` which is closer to what it actually is. `Error` was a pretty bad name for the old type tbh as it was really a full response which didn't necessarily have any errors in it!